### DISCUSSION TITLE
an err when run dpgen

### DISCUSSION THREAD

**Discussion by 1119609754** [deepmodeling/dpgen#1210] [Q&A] [answered]
I'm running dpgen in Bohrium, using the param.json file in the case and changing it to my own parameters and running it and then running through the train part will give an error and not go on to the next step. 

The error:
Traceback (most recent call last):
  File "/opt/mamba/bin/dpgen", line 8, in <module>
    sys.exit(main())
  File "/opt/mamba/lib/python3.10/site-packages/dpgen/main.py", line 185, in main
    args.func(args)
  File "/opt/mamba/lib/python3.10/site-packages/dpgen/generator/run.py", line 3642, in gen_run
    run_iter (args.PARAM, args.MACHINE)
  File "/opt/mamba/lib/python3.10/site-packages/dpgen/generator/run.py", line 3618, in run_iter
    run_model_devi (ii, jdata, mdata)
  File "/opt/mamba/lib/python3.10/site-packages/dpgen/generator/run.py", line 1556, in run_model_devi
    run_md_model_devi(iter_index,jdata,mdata)
  File "/opt/mamba/lib/python3.10/site-packages/dpgen/generator/run.py", line 1550, in run_md_model_devi
    submission.run_submission()
  File "/opt/mamba/lib/python3.10/site-packages/dpdispatcher/submission.py", line 212, in run_submission
    self.generate_jobs()
  File "/opt/mamba/lib/python3.10/site-packages/dpdispatcher/submission.py", line 394, in generate_jobs
    raise RuntimeError("submission must have at least 1 task")
RuntimeError: submission must have at least 1 task

my  param.json:
{
    "type_map": ["C"], 
    "mass_map": [12], 
    "init_data_prefix": "./data", 
    "init_data_sys": ["deepmd"], 
    "sys_configs": [
        [
            "./data/c.POSCAR.01x01x01/01.scale_pert/sys-0002/scale-1.000/00000*/POSCAR"
        ], 
        [
            "./data/c.POSCAR.01x01x01/01.scale_pert/sys-0002/scale-1.000/00001*/POSCAR"

        ]
    ],  
    "numb_models": 4, 
    "train_param": "input.json", 
    "default_training_param": {
        "model": {
            "type_map": ["C"],
            "descriptor": {
                "type": "se_a",
                "sel": [4],
                "rcut_smth": 0.5,
                "rcut": 3.0,
                "neuron": [25, 50, 100],
                "resnet_dt": true,
                "axis_neuron": 12,
                "seed": 1
            },
            "fitting_net": {
                "neuron": [120, 120, 120],
                "resnet_dt": false,
                "seed": 1
            }
        },
        "learning_rate": {
            "type": "exp",
            "start_lr": 0.001,
            "decay_steps": 1000
        },
        "loss": {
            "start_pref_e": 0.02,
            "limit_pref_e": 1,
            "start_pref_f": 1000,
            "limit_pref_f": 1,
            "start_pref_v": 0.0,
            "limit_pref_v": 0.0
        },
        "training": {
            "_set_prefix": "set",
            "stop_batch": 200000,
            
            "_batch_size": 1,
            "disp_file": "lcurve.out",
            "disp_freq": 1000,
            "numb_test": 4,
            "save_freq": 1000,
            "save_ckpt": "model.ckpt",
            "disp_training": true,
            "time_training": true,
            "profiling": false,
            "profiling_file": "timeline.json",
            "_comment": "that's all"
        }
    },
    "model_devi_dt": 0.002, 
    "model_devi_skip": 0, 
    "model_devi_f_trust_lo": 0.05, 
    "model_devi_f_trust_hi": 0.15, 
    "model_devi_clean_traj": true, 
    "model_devi_jobs": [
        {
            "sys_idx": [0], 
            "temps": [300], 
            "press": [1], 
            "trj_freq": 10, 
            "nsteps": 2000, 
            "ensemble": "nvt", 
            "_idx": "00"
        }, 
        {
            "sys_idx": [1], 
            "temps": [300], 
            "press": [1], 
            "trj_freq": 10, 
            "nsteps": 5000, 
            "ensemble": "nvt", 
            "_idx": "01"
        }
    ], 
    "ratio_failed": 0.20,
    "fp_style": "vasp",
    "shuffle_poscar": false,
    "fp_task_max": 20,
    "fp_task_min": 5,
    "fp_pp_path": "./VASP/",
    "fp_pp_files": ["POTCAR_C"],
    "fp_incar": "./VASP/INCAR_methane", 
    "_comment": "that's all"
}

**Comment by AnguseZhang** [comment, marked as answer]
You may check whether the path of `sys_configs` is correct.

**Reply by 1119609754** [reply]
thank u .
